strings/camel_case_to_snake_case.py

```diff
@@ -54,7 +54,9 @@ def camel_to_snake_case(input_str: str) -> str:
     return snake_str
 
 
-if __name__ == "__main__":
+if __name__ == "_main_":

```

Comment:
This code was also originally correct. Please look up how `__name__` works in Python.